backend/subroker/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 3.2.6 on 2021-11-12 05:33
+# Generated by Django 3.1.3 on 2021-11-18 08:33
```

Comment:
@muhwagwa maybe your django version is 3.1.3. i think it should be 3.2.6.

Reply:
aha okay i'll update my version!